src/lxc/confile.c

```diff
@@ -730,6 +810,82 @@ static int set_config_net_ipv6_gateway(const char *key, const char *value,
 	return 0;
 }
 
+static int set_config_net_ipv6_route(const char *key, const char *value,
+				       struct lxc_conf *lxc_conf, void *data)
+{
+	int ret;
+	struct lxc_netdev *netdev = data;
+	struct lxc_inet6dev *inet6dev;
+	struct lxc_list *list;
+	char *slash, *valdup, *netmask;
+
+	if (lxc_config_value_empty(value))
+		return clr_config_net_ipv6_route(key, lxc_conf, data);
+
+	if (!netdev)
+		return -1;
+
+	if (netdev->type != LXC_NET_VETH) {
+		SYSERROR("Invalid ipv6 route \"%s\", can only be used with veth network", value);
+		return -1;
+	}
+
+	inet6dev = malloc(sizeof(*inet6dev));
+	if (!inet6dev)
+		return -1;
+	memset(inet6dev, 0, sizeof(*inet6dev));
+
+	list = malloc(sizeof(*list));
+	if (!list) {
+		free(inet6dev);
+		return -1;
+	}
+
+	lxc_list_init(list);
+	list->elem = inet6dev;
+
+	valdup = strdup(value);
+	if (!valdup) {
+		free(list);
+		free(inet6dev);
+		return -1;
+	}
+
+	slash = strstr(valdup, "/");
+	if (!slash) {
+		goto out_err;
+	}
+
+	*slash = '\0';
+	netmask = slash + 1;
+
+	if (!netmask) {
```

Comment:
That can't be NULL. What you probably means is `!*netmask` or `!netmask[0]`. That should probably be:
```
if (*netmask == '\0')
```
for the sake of clarity.

Reply:
Actually,
```c
*slash = '\0';
slash++;
if (*slash == '\0')
        goto out_err;

netmask = slash;
```